e2e_test/analysis_options.yaml

```diff
@@ -0,0 +1 @@
+../analysis_options.yaml
```

Comment:
We don't need this do we?

Reply:
We do – or else (for some reason) analyzer tries to resolve the import lints relative to the (nested) package root. And... 💥